smart-contract-example/README.md

```diff
@@ -0,0 +1,90 @@
+## Smart Contract Workflow Example
+
+This example demonstrates how Kurtosis can be used for smart contract dapp development 
+by using the [`eth-network-package`](https://github.com/kurtosis-tech/eth-network-package) as a local Ethereum network. 
+The `eth-network package` can be used as a low overhead, configurable, and composable alternative to services like
+`hardhat-network`, `ganache` and `anvil`.
+
+### Setup
+
+This folder contains a typical setup for a smart contract developer looking to develop a dapp using the 
+[Hardhat](https://hardhat.org/) framework. Let's explore it's contents.
+- `contracts/` contains a few simple smart contracts for a Blackjack dapp 
+- `scripts/` contains a script to deploy a token contract to our local Ethereum network
+- `test/` contains a simple test for our token contract
+- `hardhat.config.ts` configures our Hardhat setup. 
+It allows us to configure Hardhat to use a local Ethereum network created by the `eth-network-package`.
+
+### Running the Example
+
+This assumes you have the following services installed:
+- [Kurtosis CLI](https://docs.kurtosis.com/cli/)
+- [`npx`](https://www.npmjs.com/package/npx)
+- Hardhat
+
+1. Run 
+    ```
+    kurtosis run github.com/kurtosis-tech/eth-network-package
+    ```
+    The output should look something like this
+    ```
+    ========================================== User Services ==========================================
+    UUID           Name                                           Ports                                         Status
+    bbccb71bf6db   cl-client-0-beacon                             http: 4000/tcp -> http://127.0.0.1:64256      RUNNING
+                                                                  metrics: 5054/tcp -> http://127.0.0.1:64257   
+                                                                  tcp-discovery: 9000/tcp -> 127.0.0.1:64258    
+                                                                  udp-discovery: 9000/udp -> 127.0.0.1:59023    
+    a4429f19d246   cl-client-0-validator                          http: 5042/tcp -> 127.0.0.1:64262             RUNNING
+                                                                  metrics: 5064/tcp -> http://127.0.0.1:64263   
+    77c853ab371b   el-client-0                                    engine-rpc: 8551/tcp -> 127.0.0.1:64246       RUNNING
+                                                                  rpc: 8545/tcp -> 127.0.0.1:64248              
+                                                                  tcp-discovery: 30303/tcp -> 127.0.0.1:64247   
+                                                                  udp-discovery: 30303/udp -> 127.0.0.1:58705   
+                                                                  ws: 8546/tcp -> 127.0.0.1:64249               
+    0d8469f528c9   prelaunch-data-generator-1680298734419927594   <none>                                        STOPPED
```

Comment:
What are these prelaunch data generators, why are they important/used, and why are they stopped? Would love a sentence or two about that!

Reply:
Great idea.